.github/workflows/stale.yml

```
@@ -0,0 +1,29 @@
+name: Close stale issues and PRs
+
+on:
+  schedule:
+    # run weekly on sunday
+    - cron: "0 0 * * 0"
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          days-before-issue-stale: 14
+          days-before-pr-stale: 360 # a year
+          days-before-close: -1
+          days-before-issue-close: -1
+          days-before-pr-close: -1
+          stale-issue-message: "This issue has been inactive for a while. Please add a comment or provide an update if it's still relevant."
+          stale-pr-message: "This PR has had no recent activity. Please update it if it's still relevant."
+          close-issue-message: "Closing this issue as it has been marked stale and no further activity was detected. Feel free to reopen if needed."
+          exempt-issue-labels: rfc,noslate
```

```suggestion
          exempt-issue-labels: rfc,nostale
```
Assuming this was a typo ;)